static/js/modules/helpers.js

```diff
@@ -216,6 +215,41 @@ function zeroPad(container, item, appendee) {
   });
 }
 
+function amendmentVersion(data) {
+  if (data === true) {
+    return '<i class="icon-circle--check-outline--inline--left"></i>Current version';
+  }
+  else if (data === false) {
+    return '<i class="icon-circle--clock-reverse--inline--left"></i>Past version';
+  }
+  else {
+    return 'Version unknown';
+  }
+}
+
+function amendmentVersionDescription(row) {
+  var description = '';
+  var amendment_num = 1;
+
+  // because of messy data, do not show if not e-filing or null amendment indicator
+  if (row.means_filed === 'e-file' && row.amendment_indicator !== null) {
+    if (row.amendment_indicator === 'N') {
```

Comment:
I think probably with constants. Roughly like,

```javascript
Api = {}
Api.MEANS_E_FILED = 'e-file'
Api.AMENDMENT_INDICATOR_NEW = 'N'
```

This way, the API is defined in just one place and misspellings are more likely to get caught:

```javascript
if (row.means_filed === Api.MEANS_E_FILED)
```